WhaleSpotting/react-app/src/components/MapChart.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from "react";
+import {
+    ComposableMap,
+    Geographies,
+    Geography,
+    Marker,
+    ZoomableGroup
+} from "react-simple-maps";
+import SightingApiModel from "../apiModels/SightingApiModel";
+import { Chosen } from "./Map";
+import "../styles/Map.scss";
+
+const geoUrl = "https://raw.githubusercontent.com/zcreativelabs/react-simple-maps/master/topojson-maps/world-110m.json";
+
+async function populateSightingData(): Promise<SightingApiModel[]> {
+    const response = await fetch("http://hotline.whalemuseum.org/api.json?limit=1000");
+    const response2 = await fetch("http://hotline.whalemuseum.org/api.json?limit=1000&page=2");
```

Comment:
```suggestion
    const response = await fetch("https://hotline.whalemuseum.org/api.json?limit=1000");
    const response2 = await fetch("https://hotline.whalemuseum.org/api.json?limit=1000&page=2");
```
Can we make these https pls